### ISSUE TITLE
Deferent animations for windows move on automatic resizing vs windows move on manuel resizing

### ISSUE THREAD

**Issue by godalming123** [hyprwm/Hyprland#459]
I cannot create a window bounce effect without making windows have a glitchy resize animation.
If I add
```
bezier=overshot,0.7,0.6,0.1,1.1

animations {
    enabled=1
    animation=windows,1,5,overshot,slide
   animation=windowsMove,1,5,overshot,default # the overshot animtion here cuases windows to be scaled when resized manually (see https://github.com/hyprwm/Hyprland/issues/437)
}
```
Then I get this problem https://github.com/hyprwm/Hyprland/issues/437
I think to fix this there should be 2 windowsMove suboptions;
- windowsMoveAutomatic (when windows are moved with the keyboard)

= and =
- windowsMoveManuel (when windows are move with the mouse)

This means I can have the manuel option set to disable and the autoamtic option set to my overwhoot bezier

**Comment by vaxerski** [comment]
no. The fact that they were animated in the first place was unintentional. Fixed in f36cd350ccb21260def0c02cb9de803a85c4b4b8

**Comment by vaxerski** [comment]
small patch to fix larger trees e13f2480ff0f28279d254d25711fe7d55015706f
